v8-riscv-tools/init_cross_env.sh

```diff
@@ -0,0 +1,140 @@
+    treat_warnings_as_errors=false
+    symbol_level = 0'
+ninja -C out/riscv64.native.debug -j $(nproc)
+
+# Remove obj and gen files that not needed.
+rm -rf out/riscv64.native.debug/obj
+rm -rf out/riscv64.native.debug/gen
+EOT
+
+# +++++++++++++++++++++++++++++++++++++++++
+# RISC-V QEMU
+# +++++++++++++++++++++++++++++++++++++++++
+
+cd $V8_ROOT/
+git clone https://github.com/qemu/qemu.git
+cd qemu
+# NOTE: master branch breaks current fedora image
+git checkout v5.0.0
+git submodule update -r --init -f
+./configure --target-list=riscv64-softmmu && make -j $(nproc)
+
+# optional.
+#sudo make install
+
+###########################################################
+# Deploy the Fedora Developer Rawhide on QEMU/RISCV64
+###########################################################
+
+cd $V8_ROOT/
+wget https://dl.fedoraproject.org/pub/alt/risc-v/repo/virt-builder-images/images/Fedora-Developer-Rawhide-20191123.n.0-fw_payload-uboot-qemu-virt-smode.elf
+wget https://dl.fedoraproject.org/pub/alt/risc-v/repo/virt-builder-images/images/Fedora-Developer-Rawhide-20191123.n.0-sda.raw.xz
+unxz -k Fedora-Developer-Rawhide-20191123.n.0-sda.raw.xz
+
+cat <<"EOTT"
+# NOTE: the default image size is relatively small.
+#       If you want to have a larger disk space in QEMU. run:
+
+sudo apt install libguestfs-tools -y
+truncate -r Fedora-Developer-Rawhide-*.raw expanded.raw
+truncate -s 60G expanded.raw
+
+# FIXME: we may not need sudo to run this three commands.
+sudo virt-resize -v -x --expand /dev/sda4 Fedora-Developer-Rawhide-*.raw expanded.raw
+sudo virt-filesystems --long -h --all -a expanded.raw
+sudo virt-df -h -a expanded.raw
+EOTT
+
+echo "Congrats! Now we are ready to Start QEMU."
+echo
+echo "Open a new terminal (Tab) to start QEMU"
+echo "copy these commands in the new terminal:"
+echo
+echo "export VER=20191123.n.0"
+echo "$V8_ROOT/qemu/riscv64-softmmu/qemu-system-riscv64 \\"
+echo "  -nographic \\"
+echo "  -machine virt \\"
+echo "  -smp 4 \\"
+echo "  -m 4G \\"
+echo "  -kernel Fedora-Developer-Rawhide-${VER}-fw_payload-uboot-qemu-virt-smode.elf \\"
+echo "  -object rng-random,filename=/dev/urandom,id=rng0 \\"
+echo "  -device virtio-rng-device,rng=rng0 \\"
+echo "  -device virtio-blk-device,drive=hd0 \\"
+echo "  -drive file=expanded.raw,format=raw,id=hd0 \\"
+echo "  -device virtio-net-device,netdev=usernet \\"
+echo "  -netdev user,id=usernet,hostfwd=tcp::3333-:22"
+echo
+echo "Tip: You can quit qemu by pressing 'Ctrl-a x' key sequence."
+echo "Tip: fedora forbid root password login. Either upload your pubkey into"
+echo "     ROOT/.ssh/authorized_keys or add 'PermitRootLogin=yes' in /etc/ssh/sshd_config"
+
+cat <<"EOT"
+# After you built native d8, you could run these commands to test your d8 in QEMU
+# please note the 3333 port, it must match your QEMU setting above.
+scp -r -P 3333 $V8_ROOT/v8/out/riscv64.native.debug $V8_ROOT/v8/tools $V8_ROOT/v8/test root@localhost:~/
+ssh -p 3333 root@localhost python2 ./tools/run-tests.py \
```

Comment:
Since the list of test suites keeps on changing, it is better that we all use `v8-riscv-tools/test-riscv.sh` to run the complete suite of tests.

Right now `v8-riscv-tools/test-riscv.sh` defaults to pick the simulated build. Could you make it to overwrite the default build if  a command-line option `--outdir` is specified? This way we can use the same script to run tests for all builds.

run all the tests as the test list may be changing.

Reply:
ok. I'd update the test-riscv.sh